@gasinvein 

```
QMake version 3.1
Using Qt version 5.15.3 in /usr/lib/x86_64-linux-gnu
CMake Deprecation Warning at CMakeLists.txt:54 (cmake_policy):
  The OLD behavior for policy CMP0071 will be removed from a future version
  of CMake.

  The cmake-policies(7) manual explains that the OLD behaviors of all
  policies are deprecated and that a policy should be set to OLD only under
  specific short-term circumstances.  Projects should be ported to the NEW
  behavior and not rely on setting a policy to OLD.


CMake Error at /app/lib/x86_64-linux-gnu/cmake/Qt5WebEngineCore/Qt5WebEngineCoreConfig.cmake:26 (message):
  The imported target "Qt5::WebEngineCore" references the file

     "/app/lib/x86_64-linux-gnu/libQt5WebEngineCore.so.5.15.3"

  but this file does not exist.  Possible reasons include:

  * The file was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and contained

     "/app/lib/x86_64-linux-gnu/cmake/Qt5WebEngineCore/Qt5WebEngineCoreConfig.cmake"

  but not all the files it references.

Call Stack (most recent call first):
  /app/lib/x86_64-linux-gnu/cmake/Qt5WebEngineCore/Qt5WebEngineCoreConfig.cmake:44 (_qt5_WebEngineCore_check_file_exists)
  /app/lib/x86_64-linux-gnu/cmake/Qt5WebEngineCore/Qt5WebEngineCoreConfig.cmake:213 (_populate_WebEngineCore_target_properties)
  /app/lib/x86_64-linux-gnu/cmake/Qt5WebEngine/Qt5WebEngineConfig.cmake:111 (find_package)
  build/FindQt5.cmake:42 (find_package)
  CMakeLists.txt:84 (include)


-- Configuring incomplete, errors occurred!
See also "/run/build/musescore/CMakeFiles/CMakeOutput.log".
Error: module musescore: Child process exited with code 1
```

I'm guessing that this is probably because I should only change the `$LIB_VERSION_STRING`, like `set(Qt5WebEngineCore_VERSION_STRING 5.15.3)`.  
Now that I'm looking at this, I believe I'm probably even breaking headers paths, so I revisit this later and send a proper patch.

--- a/io.qt.qtwebengine.BaseApp.json
+++ b/io.qt.qtwebengine.BaseApp.json
@@ -188,6 +188,7 @@
                 }
             ],
             "build-commands": [
+                "ARCH_TRIPLE=$(gcc --print-multiarch); qtver=$(qmake -query QT_VERSION); qtwebver=$(grep -o '5[0-9.]*' ${FLATPAK_DEST}/lib/${ARCH_TRIPLE}/cmake/Qt5WebEngine/Qt5WebEngineConfigVersion.cmake); lesser_ver=$(echo -e \"${qtver}\n${qtwebver}\" | sort -V | head -1); sed -i \"s/$qtwebver \\(\\${_Qt5WebEngine\\)/$lesser_ver \\1/\" ${FLATPAK_DEST}/lib/${ARCH_TRIPLE}/cmake/Qt5WebEngine*/Qt5WebEngine*Config.cmake",
                 "sed -e 's@PATHS \"${CMAKE_CURRENT_LIST_DIR}/..\" NO_DEFAULT_PATH@PATHS \"${CMAKE_CURRENT_LIST_DIR}/..\" \"/usr/lib/${CMAKE_CXX_LIBRARY_ARCHITECTURE}/cmake/\" NO_DEFAULT_PATH@' -i /app/lib/*/cmake/*/*Config.cmake",
                 "sed -e 's@$$QT_MODULE_LIB_BASE@/app/lib@' -i /app/lib/mkspecs/modules/*.pri",
                 "mv /app/lib/libexec/QtWebEngineProcess -t /app/bin",

